steelthread-0/walkthrough.md

```diff
@@ -15,123 +18,234 @@ The installation has already been performed on an OpenShift cluster as per the i
 
-```
-Deploy the applications
+Edit the `messaging-service` configmap, setting the `MESSAGING_SERVICE_PASSWORD` & `MESSAGING_SERVICE_USER` to the values from the EnMasse binding secret.
 
 ```
-oc new-app --template=nodejs-messaging-work-queue-frontend
-oc new-app --template=nodejs-messaging-work-queue-worker
+oc edit cm messaging-service -n eval
 ```
 
-Edit the messaging-service configmap
+Restart the booster services
 
 ```
-oc edit cm messaging-service -n eval:
+oc rollout latest nodejs-messaging-work-queue-frontend -n eval
 ```
 
-Add in the values from the secret from the Enmasse binding secret
-
-Set the following key value pairs
+In the log of the application you should see something like:
 
 ```
-MESSAGING_SERVICE_HOST = <value from messagingHost>
-MESSAGING_SERVICE_PASSWORD = <value from password>
-MESSAGING_SERVICE_USER= <value from username>
+frontend-nodejs-aad7: Connected to AMQP messaging service at messaging.enmasse-eval.svc:5672
 ```
 
-Restart the booster services
+# Setup Enmasse connection in Fuse Ignite
 
-```
-oc rollout latest nodejs-messaging-work-queue-frontend
-oc rollout latest nodejs-messaging-work-queue-worker
-```
+You will need the `username` and `password` credentials again for this section from the EnMasse binding secret (`enmasse-standard-xxxxx-credentials-yyyyy`)
+
+- Open the Fuse Ignite console. You will find the route in the OpenShift Web Console overview screen for the `eval` project.
+- Login in to the Fuse Ignite console using the evals@example.com user 
+- Select `Connections` and `Create connection`
+- Find the `AMQP Message Broker` connector and select it. 
+- Fill in the Connection URI Field with the following  ```amqp://messaging.enmasse-eval.svc:5672?amqp.saslMechanisms=PLAIN```
+- Fill in the username and password fields with the `username` and `password` values from the EnMasse binding secret.
+- Choose `Disable` for the Check Certificates dropdown
+- Click Validate 
+- Click the `Next` button at the top right
+- Add a connection name ```enmasse```
+- Click the `Create` button at the top right
 
-# Setup HTTP connection to the messaging booster in Fuse Ignite
-- Open the Fuse ignite console and open the connections again. 
-- Click create connection
-- Find the HTTP connection
-- Set the base URL as the url exposed from the messaging front end application
-- Click validate to ensure Fuse Ignite can reach the endpoint
-- Click next 
-- Give the connection a name (booster)
-- Click create
+# Setup HTTP connection to the CRUD REST booster in Fuse Ignite
+
+- Stay in the Fuse Ignite console for this section
+- Select `Connections` and `Create connection`
+- Find the `HTTP Connector` and select it. 
+- Set the base URL as the url exposed from the CRUD REST booster application that we created earlier (You can get this using `oc get route spring-boot-rest-http-crud -n eval`)
+- Click `Validate` to ensure Fuse Ignite can reach the endpoint
+- Click `Next` 
+- Give the connection a name (crud booster)
```

Comment:
To stay consistent with the previous block we should state: Add a connection name ```enmasse``` instead.